packages/protocol/test/governance/validators.ts

```diff
@@ -1091,6 +1098,7 @@ contract('Validators', (accounts: string[]) => {
           })
 
           it('should set the validator ecdsa public key', async () => {
+            await registry.setAddressFor(CeloContractName.Accounts, accountsInstance.address)
```

Comment:
why do you need this?

Reply:
`getValidator` calls `getAccount().getValidatorSigner` account.address needs to be contract, not wallet

Reply:
https://github.com/celo-org/celo-monorepo/pull/1997/files/f308cff7a9aa1a716337f18747382358a7984b2f#diff-cee92562946fb3c6fc71ec4d162da770R1087